tests/test_postgres_native.nit

```diff
@@ -0,0 +1,67 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Copyright 2016 Guilherme Mansur <guilhermerpmansur@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module test_postgres_native
+
+import postgresql::native_postgres
+
+#var conninfo = "dbname=testdb user=memo hostaddr=127.0.0.1 port=5432"
+var conninfo = "dbname=postgres"
+var create_req = "CREATE TABLE IF NOT EXISTS animals (aname TEXT PRIMARY KEY, class TEXT NOT NULL, sex INTEGER)"
+var insert_req_1 = "INSERT INTO animals VALUES('Lioness', 'mammal', 1)"
+var insert_req_2 = "INSERT INTO animals VALUES('Snake', 'reptile', 0)"
+var select_req = "SELECT * FROM animals"
+var delete_req_1 = "DELETE FROM animals WHERE aname = 'Lioness'"
+var delete_req_2 = "DELETE FROM animals WHERE aname = 'Snake'"
+var delete_req_3 = "DELETE FROM animals WHERE aname = 'Frog'"
+var prepare = "INSERT INTO animals(aname, class, sex) VALUES ($1, $2, $3)"
+var drop_table = "DROP TABLE animals"
+var db = new NativePostgres.pq_connectdb(conninfo)
+assert postgres_open: db.pq_status.is_ok
+
+var status = db.pq_exec(create_req)
+assert postgres_create_table: status.is_ok
+
+status = db.pq_exec(insert_req_1)
+assert postgres_insert_1: status.is_ok
+
+status = db.pq_exec(insert_req_2)
+assert postgres_insert_2: status.is_ok
+
+status = db.pq_exec(select_req)
```

Comment:
If I'm not mistaken `ExecStatusType` only tells the type of the data, returning only the result of `PQresultStatus` loses the data itself. Have you wrapped a service to get the first line of `SELECT * FROM animals`?